aesara/tensor/rewriting/elemwise.py

```diff
@@ -896,16 +896,16 @@ def print_profile(cls, stream, prof, level=0):
         print(blanc, " time_toposort", prof[7], file=stream)
 
 
+fuse_seqopt = SequenceDB()
```

Comment:
This was potentially buggy, as `fuse_seqopt` was always being imported by the other module, even though it's creation was conditional on a config flag.